virtualization/windowscontainers/container-networking/network-isolation-security.md

```diff
@@ -24,26 +24,47 @@ Each container endpoint is placed in its own __network compartment__ which is an
 Get-NetCompartment
 ```
 
-## Windows Firewall Security
-
-The Windows Firewall is used to enforce network security through port ACLs.
+## Network Security
+Depending on which container and network driver is used, Windows Firewall or VFP is used to enforce network security through port ACLs.
 
 > NOTE: Make sure your environment satisfies these *required* [prerequisites](https://docs.docker.com/network/overlay/#operations-for-all-overlay-networks) for creating overlay networks.
 
 
 ### Windows Server containers
-These use the Windows hosts' firewall (enlightened with network compartments.) 
+These use the Windows hosts' firewall (enlightened with network namespaces)
   * Default Outbound: ALLOW ALL
-  * Default Inbound: DENY ALL unsolicited
-    * Except "common" (e.g. DHCP, DNS, ICMP, etc.) network traffic
+  * Default Inbound: ALLOW ALL "common" (e.g. TCP,DHCP, DNS, ICMP, etc.) unsolicited network traffic
+    * DENY ALL "uncommon" network traffic
     * Create inbound ALLOW rule in response to ``docker run -p`` (port forwarding)
 
+
 ### Hyper-V containers
 Hyper-V containers have their own isolated kernel and hence run their own instance of Windows Firewall with the following configuration:
   * Default ALLOW ALL in Windows Firewall
 
 Note that in general, all container endpoints attached to an overlay network have an:
-  *  ALLOW ALL rule created.  
+  *  ALLOW ALL rule created.
 
```

Comment:
Even though the Windows Firewall in the Utility VM is set to default ALLOW ALL, we do have default DENY rules in VFP at a vSwitch level, correct? I hope?